streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignor.java

```diff
@@ -354,6 +354,11 @@ public Subscription subscription(Set<String> topics) {
             }
         } while (numPartitionsNeeded);
 
+        // ensure the co-partitioning topics within the group have the same number of partitions,
```

Comment:
Grammar?  You mean "ensure the co-partitioning [of] topics within the group so that they have the same..."?